src/mysky.ts

```diff
@@ -219,6 +261,30 @@ export class MySky {
     return publicKey;
   }
 
+  /**
+   * verifyMessageSignature verifies the signature for the message and given
+   * public key and returns a boolean that indicates whether the verification
+   * succeeded.
+   *
+   * @param message - the original message that was signed
+   * @param signature - the signature
+   * @param publicKey - the public key
+   * @returns boolean that indicates whether the verification succeeded
+   */
+  async verifyMessageSignature(message: Uint8Array, signature: Uint8Array, publicKey: string): Promise<boolean> {
+    // transform the public key to a by array
+    const publicKeyBytes = fromHexString(publicKey);
+    if (!publicKeyBytes) {
```

Comment:
Same as above - it would be nice to verify that the key is not only truthy by also has the expected length of 32 bytes.

Reply:
same as above